WebUI/src/Heatmap.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import "./Heatmap.css";
+import Prism from "prismjs";
+
+export default function (view: any) {
```

Comment:
Thanks for converting my mess to TS! Looks good, as far as I can tell.

Reply:
Glad you like it. :-)